Source/WebCore/platform/RegistrableDomain.h

```diff
@@ -91,7 +91,7 @@ class RegistrableDomain {
     
     static RegistrableDomain uncheckedCreateFromHost(const String& host)
     {
-        auto registrableDomain = PublicSuffixStore::singleton().topPrivatelyControlledDomain(host);
+        auto registrableDomain = PublicSuffixStore::singleton().registrableDomain(URL { host });
```

Comment:
This won't work. A host cannot be parsed a URL. You could maybe prefix it with `https://`, but that does not seem great. Perhaps callers of this `uncheckedCreateFromHost()` can use something else? If not, it might need its own API.